subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -493,8 +493,9 @@ img_mgmt_upload(struct mgmt_ctxt *ctxt)
 			last = true;
 		}
 
-		rc = img_mgmt_impl_write_image_data(req.off, req.img_data, action.write_bytes,
-						    last);
+		//rc = img_mgmt_impl_write_image_data(req.off, req.img_data, action.write_bytes,
+		//				    last);
+		rc = 0;
```

Comment:
Leftover code from tests

Reply:
Actually I have left it intentionally here, for the draft, because it allows to give flash some slack during tests and removes flash operations delay; when comparing performance against non-re-assembly code, in image upload scenario, this should be applied to remove flash operations impact on testing.